.github/workflows/docker.yml

```
@@ -0,0 +1,21 @@
+name: Docker
+on:
+  pull_request:
+    paths:
+      - .github/workflows/docker.yml
+      - Dockerfile
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Build Docker image
+        run: docker build -t brew-wheezy-build-box .
```

```suggestion
        run: docker build -t homebrew/debian7.11 .
```
To reflect the naming scheme of `homebrew/ubuntu16.04` at `https://hub.docker.com/u/homebrew`

---

I would say debian7 is enough.

---

I would give it a specific name. This image is used only for patchelf and binutils builds, and contains a hacked git version. This image should not be used elsewhere.

We can have a different, clean, Debian 7 image somewhere else if you want.

---

One Debian 7 image is definitely enough. =)

---

A hacked Git version may be the only way to get Homebrew on Debian 7. I'm personally fine with and prefer calling this image `homebrew/debian7`.